lib/Cro/Tools/Web.pm6

```diff
@@ -32,6 +32,9 @@ sub web(Str $host, Int $port, $runner) is export {
         get -> {
             content 'text/html', %?RESOURCES<web/index.html>.slurp;
         }
+        get -> Str $app-route where "stub" | "logs" {
+            content 'text/html', %?RESOURCES<web/index.html>.slurp;
+        }
```

Comment:
We could also lose the `where` here and make it handle any app routes we add in the future, since the router prefers longest literal.